src/components/dialog/Dialog.tsx

```diff
@@ -93,9 +97,10 @@ const Dialog: GenericComponent<DialogProps> = ({
       <DialogBase.Footer
         display="flex"
         justifyContent={leftAction ? 'space-between' : 'flex-end'}
+        alignItems="center"
         className={cx({ [theme['scroll-shadow']]: !reachedScrollEnd && showScrollShadow })}
       >
-        {leftAction && <Button {...leftAction} />}
+        {leftAction && (leftAction.isToggle ? <Toggle {...leftAction} /> : <Button {...leftAction} />)}
```

Comment:
Q: I guess typescript will complain about this, right? Because `isToggle` is not a valid prop?